graylog-storage-elasticsearch7/src/test/java/org/graylog/storage/elasticsearch7/testing/ElasticsearchInstanceSmallES7.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+package org.graylog.storage.elasticsearch7.testing;
+
+import org.graylog2.storage.SearchVersion;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+
+public class ElasticsearchInstanceSmallES7 extends ElasticsearchInstanceES7 {
```

Comment:
Instead of creating a new subclass, we could also extend the `ElasticsearchInstanceES7` (same for ES6 of course) class to accept a parameter that allow customizing the memory limits. That would avoid a bit of boiler plate and makes it a bit cleaner, I think.

Reply:
Agreed. I didn't like to having to duplicate this entire chain of static methods either.